pvlib/transformer.py

```diff
@@ -0,0 +1,115 @@
+"""
+This module contains functions for transformer modeling.
+
+Transformer models calculate AC power output at a different voltage from the
+voltage at the AC input terminals.
+"""
+
+import numpy as np
+
+
+def simple_efficiency(
+        input_power, no_load_loss, load_loss, transformer_rating
+):
+    r'''
+    Calculate the energy at the output terminal of the transformer
+    after taking into account efficiency using a simple calculation.
+
+    The equation used in this function can be derived from [1]_.
+
+    First, assume that the load loss is proportional to the square of output
```

Comment:
We may want to move the material from here to "Parameters" to after "Parameters" under a "Notes" heading. Like the [martin_ruiz](https://pvlib-python--2053.org.readthedocs.build/en/2053/reference/generated/pvlib.iam.martin_ruiz.html#pvlib.iam.martin_ruiz) docstring.

Reply:
Updated docstring to include a new notes heading, following the martin and ruiz docstring example